public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,78 @@
+# Setup Google Drive remote
+
+To setup Google Drive remote you should
+[create and configure Google Project](#create-and-configure-google-project) if
+you don't have one yet.
+
+## Create and Configure Google Project
+
+Since Google Drive has tight API usage quotas, creation and configuration of
+your own `Google Project` is required:
+
+1.  Log into the [Google Cloud Platform](https://console.developers.google.com)
+    account.
+2.  Create `New Project` or select available one.
+3.  Click `ENABLE APIS AND SERVICES` and search for `drive` to enable
+    `Google Drive API` from search results.
+4.  Navigate to
+    [All Credentials](https://console.developers.google.com/apis/credentials)
+    page and click `Create Credentials` to select `OAuth client ID`. It might
+    ask you to setup a product name on the consent screen.
+5.  Select `Other` for `Application type` and click `Create` to proceed with
+    default `Name`.
+6.  `client id` and `client secret` should be showed to you. Use them for
+    further DVC's configuration.
+
+```dvc
+$ dvc remote add myremote gdrive://root/my-dvc-root
```

Comment:
add a comment something along this line - "see below to understand how to create a path ..."